Solutions/PaloAltoPrismaCloud/Analytic Rules/PaloAltoPrismaCloudHighRiskScoreAlert.yaml

```diff
@@ -0,0 +1,30 @@
+id: 617b02d8-0f47-4f3c-afed-1926a45e7b28
+name: Palo Alto Prisma Cloud - Maximum risk score alert

```

Comment:
@vu-socprime ,"PaloAltoPrismaCloudMaxRiskScoreAlert.yaml" analytics rule having same name. Could you please update name of this Analytics rule to"Palo Alto Prisma Cloud - **High** risk score alert".   Thanks!

Reply:
Updated in PR https://github.com/Azure/Azure-Sentinel/pull/2116

Reply:
@vu-socprime , Two hunting queries are failing, I have given comment for same, can you please fix those in same PR. Thanks!